internal/crypto/key_derivation_testing.go

```diff
@@ -12,14 +12,18 @@ import (
 const (
 	// DefaultKeyDerivationAlgorithm is the key derivation algorithm for new configurations.
 	DefaultKeyDerivationAlgorithm = "testing-only-insecure"
+	DefaultHashVersion            = 99
 
 	// MasterKeyLength describes the length of the master key.
 	MasterKeyLength = 32
 
-	V1SaltLength    = 32
-	HashVersion1    = 1 // this translates to Scrypt KeyDerivationAlgorithm
-	ScryptAlgorithm = "scrypt-65536-8-1"
-	Pbkdf2Algorithm = "pbkdf2"
+	V1SaltLength = 32
+
+	ScryptAlgorithm   = "scrypt-65536-8-1"
+	ScryptHashVersion = 1
```

Comment:
`ScryptHashVersion` & `Pbkdf2HashVersion` below should be defined in the `internal/user` package.